### PR TITLE
chore(website): migrate content pages for getting started, improve layout and other things

### PR DESCRIPTION
**Pull request by emmenko**
The new pages have been copied over from our current docs website. I mostly fixed the links and the rendering of the images, as well as some typos.

### CODE REVIEW

File: website/content/getting-started/deployment.mdx [hunk 1]
```diff
@@ -0,0 +1,277 @@
+---
+id: getting-started-deployment
+title: Deployment - Getting started
+permalink: /getting-started/deployment
+---
+
+import { ContentNotification } from '@commercetools-frontend/ui-kit';
+
+<ContentNotification type="info">
```
Comment (emmenko) [inline website/content/getting-started/deployment.mdx]: Using this component is a test, to see how it looks. If we do decide to use that, we would need to setup theming in order to override some of the styles.

File: website/content/getting-started/deployment.mdx [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+
+## Prepare the application for production use
+
+In order to run the application in `production` mode, we need to **build** it using the following command:
+
+```
+$ yarn build
+```
+
+![Production build](/assets/custom-applications/mc-dev-yarn-build.png)
```
Comment (emmenko) [inline website/content/getting-started/deployment.mdx]: Images now can be rendered like this. Under the hood, gatsby will transform this into an optimized image. Images used for content pages should be defined in `./static/assets` folder.

File: website/gatsby-config.js [hunk 3]
```diff
@@ -9,8 +9,12 @@ module.exports = {
       {
         label: 'Getting started',
         subgroup: [
-          { label: 'Installation', linkTo: '/getting-started/installation' },
+          { label: 'Introduction', linkTo: '/getting-started/introduction' },
```
Comment (emmenko) [inline website/gatsby-config.js]: I'm still not 100% sure about this structure/grouping, will just keep it like this for now though

File: website/gatsby-config.js [hunk 4]
```diff
@@ -27,54 +31,50 @@ module.exports = {
...
-          'roboto mono',
-          // https://design.google/library/spectral-new-screen-first-typeface/
-          // 'Spectral:400,700',
-          // https://design.google/library/choosing-web-fonts-beginners-guide/
-          // 'Libre+Baskerville:400,400i,700',
-          'Raleway:400,400i,700,700i',
-        ],
-      },
-    },
+    /**
```
Comment (emmenko) [inline website/gatsby-config.js]: I mostly rearranged the plugins and added some comments

File: website/gatsby-config.js [hunk 5]
```diff
@@ -83,8 +83,54 @@ module.exports = {
...
+    // For querying MDX
+    {
+      resolve: 'gatsby-mdx',
+      options: {
+        extensions: [
+          '.mdx',
+          // ".md"
+        ],
+        remarkPlugins: [require('remark-slug'), require('remark-emoji')],
+        gatsbyRemarkPlugins: [
```
Comment (emmenko) [inline website/gatsby-config.js]: Those plugins are newly added, which are necessary for rendering optimized images from markdown pages

File: website/src/components/markdown-layout.js [hunk 6]
```diff
@@ -13,7 +13,8 @@ const TypographyPage = styled.div`
   font-family: 'Raleway', sans-serif;
   font-weight: 400;
   line-height: 1.45;
-  padding: ${customProperties.spacingM};
+  padding: ${customProperties.spacingL};
+  max-width: 740px;
```
Comment (emmenko) [inline website/src/components/markdown-layout.js]: This ensures that the content stays within a reasonable size. This width is the same of what Medium uses

**Comment by tdeekens** [comment]
Looks super nice. Clicked around the deploy preview a bit.

File: website/gatsby-config.js [hunk 7]
```diff
@@ -83,8 +83,54 @@ module.exports = {
...
+    // For querying images
     `gatsby-transformer-sharp`,
+
+    // For querying MDX
+    {
+      resolve: 'gatsby-mdx',
+      options: {
+        extensions: [
+          '.mdx',
+          // ".md"
```
Comment (tdeekens) [inline website/gatsby-config.js]: ?

**Pull request merged**